Makefile

```diff
@@ -209,6 +209,7 @@ test-python-universal-postgres-online:
 test-python-universal-cassandra:
 	PYTHONPATH='.' \
 	FULL_REPO_CONFIGS_MODULE=sdk.python.feast.infra.online_stores.contrib.cassandra_repo_configuration \
+	PYTEST_PLUGINS=sdk.python.tests.integration.feature_repos.universal.online_store \
```

Comment:
is this right? seems like it should be using the cassandra plugins?

Reply:
fixed

Reply:
It works with the module around the online stores but I can also just plugin the actual file